src/Uno.Toolkit.UI/Controls/TabBar/TabBarItem.Properties.cs

```diff
@@ -29,26 +27,61 @@ partial class TabBarItem
 			DependencyProperty.Register(nameof(Icon), typeof(IconElement), typeof(TabBarItem), new PropertyMetadata(null, OnPropertyChanged));
 		#endregion
 
+		// UNO TODO: Deprecate and remove BadgeVisibility and BadgeValue properties and use InfoBadge instead
 		#region BadgeVisibility
```

Comment:
@kazo0, @Xiaoy312, @MartinZikmund 
Can we have something like [Obsolete("This property is deprecated. Use `InfoBadge` instead.", true)] for `BadgeVisibility` and `BadgeValue` here ?

Reply:
Related issue for reference: https://github.com/unoplatform/uno.csharpmarkup/issues/741
@morning4coffe-dev like we talked about, you add the [Obsolete] like we should have but comment it with the link of the related uno issue.